scripts/epicsArchChecker

```diff
@@ -96,7 +105,7 @@ def read_file(filename):
                 if line[0] == '<':
                     if '/' not in filename:
                         subfilename = line[1:].lstrip().strip()
-                        (subentries, subextraKeys, subnoKeyPVs) =\
+                        (filesMissing, subentries, subextraKeys, subnoKeyPVs) =\
```

Comment:
`filesMissing` is set multiple times, so is it possible for it to override itself and give the wrong result?

For example, take the following epicsArch file, where all files are present except for `notafile`:

```
real_file1
real_file2
notafile
real_file3
```

I think `filesMissing` would be set to `True` on one of the lines and then back to `False` by the end.

Possibly this should be `or`-ed together with the previous result at each line much like how the other return values are accumulated in arrays

```
(subfilesMissing, ...) = read_file(...)
filesMissing = filesMissing or subfilesMissing
```